### DISCUSSION TITLE
Is external storage not supported?

### DISCUSSION THREAD

**Discussion by Byter3** [nextcloud/recognize#917] [Q&A] [answered]
Hi,
I made a new AIO instance and pluged a NAS into it via SMB. recognize labeled every imega what is inside the AIO. Such as the Talk folder, but did nothing for the external folders. So, is external storage not supported?

**Comment by backupandforkacc** [comment]
Same for me. Is there any workaround?

**Comment by backupandforkacc** [comment, marked as answer]
External storage (other nextcloud instance) now seems to be working.

**Comment by marcelklehr** [comment]
> So, is external storage not supported?

External storage is supported.There may be problems with files getting added to external storage outside of nextcloud. Is that what is happening on your instance?
